### PR TITLE
added-2-new-themes-used-for-jettwave.com

### PR DESCRIPTION
**Pull request by ryanmogk**
Created two new themes for a project I'm working on. Figured I'd add them for others to use. 👍

### CODE REVIEW

File: packages/prism-react-renderer/src/index.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { Prism } from "./prism-langs"
-import * as themes from "./themes"
+import * as themes from "./themes/index"
```
Comment (ryanmogk) [inline packages/prism-react-renderer/src/index.ts:2]: This auto-updated. Will want to change that back

File: packages/prism-react-renderer/src/themes/jettwaveDark.ts [hunk 2]
```diff
@@ -0,0 +1,105 @@
+// light version of code viewer styles built for https://jettwave.com
```
Comment (ryanmogk) [inline packages/prism-react-renderer/src/themes/jettwaveDark.ts:1]: dark version*

**Comment by carloskelly13** [comment]
Hey thanks for adding these! I'd be happy to merge this In once we drop the package-lock.json (we use pnpm for this mono repo) and fixing the import path.

**Comment by ryanmogk** [comment]
> Added the changeset & got rid of the lock👍

**Review by carloskelly13 — APPROVED** [review]
They look great. Thank you!

**Pull request merged**